tensorboard/webapp/app_routing/route_config.ts

```diff
@@ -441,7 +441,8 @@ export class RouteConfigs {
         deepLinkProvider: definition.deepLinkProvider ?? null,
         pathname: definition.path,
         params: {},
-        originateFromRedirection: wasRedirected,
+        originateFromRedirection: true,
```

Comment:
This was an intentional change since this is handling the case where no route definition matched current url and we are "redirecting" to the default route. Also, hard coding `originateFromRedirection: true` keeps type very simple and clear.